test/integration/patients/sidebar/filter-sidebar.js

```diff
@@ -1343,5 +1343,62 @@ context('filter sidebar', function() {
       .get('@filtersSidebar')
       .should('not.exist');
   });
+
+  specify('states sorted by sequence value', function() {
+    cy
+      .routeStates(fx => {
+        fx.data = fx.data.slice(0, 3);
+
+        fx.data[0].attributes.name = 'Second In Sequence';
+        fx.data[0].attributes.sequence = 200;
+        fx.data[0].attributes.status = 'queued';
+
+        fx.data[1].attributes.name = 'Third In Sequence';
+        fx.data[1].attributes.sequence = 300;
+        fx.data[1].attributes.status = 'queued';
+
+        fx.data[2].attributes.name = 'First In Sequence';
+        fx.data[2].attributes.sequence = 100;
+        fx.data[2].attributes.status = 'queued';
+
+        return fx;
+      })
+      .routeWorkspaces(fx => {
+        fx.data[0].relationships.states.data = [
+          { id: '22222', type: 'states' },
+          { id: '33333', type: 'states' },
+          { id: '55555', type: 'states' },
+        ];
+
+        return fx;
+      })
```

Comment:
Without this `routeWorkspaces()`, this happens:

![Screenshot 2023-08-29 at 2 04 28 PM](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/f1970d63-4787-4ad8-a23d-d97034191c11)

I think because the states set on the workspace are overriding things in the test.